<launch>
    <!-- 0 -> statinary.bag; 1 -> with_gripper_perpendicular.bag; 2 -> with_gripper_parallel.bag -->
    <arg name="bag_file" default="0" />
    <arg name="use_eval_rope" default="true" />

    <node name="trackdlo" pkg="trackdlo" type="gltp" output="screen">
        <!-- 0 -> statinary.bag; 1 -> with_gripper_perpendicular.bag -->
        <param name="bag_file" value="$(arg bag_file)" />

        <param name="beta" value="1" />
        <param name="lambda" value="1" />
        <param name="lle_weight" value="1" />
        <param name="mu" value="0.05" />
        <param name="max_iter" value="50" />
        <param name="tol" value="0.00001" />
        <param name="include_lle" value="true" />
        <param name="use_prev_sigma2" value="true" />
        <param name="use_eval_rope" value="$(arg use_eval_rope)" />
        <param name="downsample_leaf_size" value="0.005" />
        <param name="num_of_nodes" value="40" />
    </node>

</launch>